modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java

```diff
@@ -551,9 +551,10 @@ protected void validateFile(
 			"layout-prototype", "layout-set", "layout-set-prototype"
 		};
 
-		Stream<String> stream = Stream.of(expectedLARTypes);
+		expectedLARTypes = ArrayUtil.filter(
```

Comment:
@JanisZhang Please try to simply this. It seems we always have these 3 elements in this array. You can either hard code them in an if-statement, or put them in a static unmodifiable hashset so that you can just do "set.contains".